yum/pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <!-- note: the *bucket* is the root of the repo -->
-        <yumrepo.s3RepositoryPath>s3://nexus-private-artifacts/emodb/yum-repo</yumrepo.s3RepositoryPath>
+        <yumrepo.s3RepositoryPath>s3://bv-emodb-artifacts/emodb/yum-repo</yumrepo.s3RepositoryPath>
```

Comment:
Done.
Currently Jenkins job was updated with the old bucket. The new bucket will be set when this and all related PRs will be merged.